students/olszewski_bartosz/lesson_05_lists/greater_than_neighbours.py

```diff
@@ -0,0 +1,6 @@
+a = input().split()
+b = 0
+for i in range(1, len(a) - 1):
+    if a[i - 1] < a[i] and a[i + 1] < a[i]:
```

Comment:
It might be shortened to:
if a[i - 1] < a[i] > a[i + 1]:

Anyway, the current implementation is also correct